src/main.rs

```diff
@@ -47,7 +45,9 @@ fn run_floki_from_args(args: &Cli) -> Result<(), Error> {
     debug!("Selected configuration file: {:?}", &environ.config_file);
 
     let config = FlokiConfig::from_file(&environ.config_file)?;
-    verify_command(args.local, &config)?;
+    if args.local {
+        eprintln!("-l/--local is deprecated and may be removed in a future release");
```

Comment:
You should be able to

```
warn!(...)
```

Rather than `eprintln!`. There's one place we can't do that, and that's when the logger is guaranteed to be configured, but that's only right at the start.

Reply:
I did eprintln as floki doesn't log warnings unless you pass -v.

I could change it to warn, and also change the default logging level to warn?

Reply:
Ah yeah I see, I probably should have set it to warn by default. Feel free to fix that - should be 1 line in `main.rs` in `configure_logging` (change the setting for verbosity 0).

I just reworked that a couple of weeks ago - should have set it to at least warn then.

Reply:
(Probably makes sense as a separate commit - but you can attach it to this PR)